lib/mpl_toolkits/axes_grid1/axes_grid.py

```diff
@@ -358,6 +358,11 @@ def __init__(self, fig,
         cbar_location : {"left", "right", "bottom", "top"}, default: "right"
         cbar_pad : float, default: None
             Padding between the image axes and the colorbar axes.
+
+            .. versionchanged:: 3.10
+                `cbar_mode="single"` no longer adds the `axes_pad` between the axes and
+                the colorbar if the `cbar_location` is `"left"` or `"bottom"`
```

Comment:
```suggestion
                ``cbar_mode="single"`` no longer adds *axes_pad* between the axes
                and the colorbar if the *cbar_location* is "left" or "bottom".
```
ReST formatting and some conventions from https://matplotlib.org/devdocs/devel/document.html#formatting-conventions.